components/DetailPageSidebar/detail-page-sidebar.scss

```diff
@@ -0,0 +1,131 @@
+@import '~carbon-components/scss/globals/scss/styles.scss';
+
+@include exports('detail-page-sidebar') {
+  .bx--detail-page-sidebar__container { 
+    width: 100%;
+    margin: 0 5% 0 0; 
+  }
+  .bx--detail-page-sidebar__container .tag-description,
+  .bx--detail-page-sidebar__container .details-description {
+      font-size: 0.875rem;
+      line-height: 1.65;
+      font-weight: 400;
+      word-wrap: break-word; 
+  }
+  .bx--detail-page-sidebar__container .tag-description {
+      margin-bottom: 2em; 
+    }
+  .bx--detail-page-sidebar__container .tag-description > span {
+        font-weight: 600; 
+  }
+    @media (max-width: 824.999px) {
+      .bx--detail-page-sidebar__container .description-label {
+        padding-top: 0;
+        padding-bottom: 0;
+        overflow: visible;
+        color: #5aaafa;
+        font-size: 0.875rem;
+        font-weight: 600;
+        line-height: 1.65;
+        margin: 0; }
+        .bx--global-light-ui .bx--detail-page-sidebar__container .description-label {
+          color: #3d70b2; }
+      .bx--detail-page-sidebar__container .description-tag-docs-link,
+      .detail-page-pieces .detail-page-sidebar__container .description-tag-terms-link {
+        display: block; } 
+  }
+  @media (min-width: 825px) {
+      .bx--detail-page-sidebar__container {
+        width: 40%; }
+        .bx--detail-page-sidebar__container .inline-notification {
+          display: none; } }
+    @media (min-width: 825px) and (min-width: 1000px) {
```

Comment:
this media query is a little strange here - generally you have a min-width and a max-width if you're pairing. Otherwise this says the same thing